keyless/pepper/common/src/account_recovery_db.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{Deserialize, Serialize};
+use aptos_crypto::compat::Sha3_256;
+use ed25519_dalek::Digest;
+
+/// The schema used in the account recovery DB.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct AccountRecoveryDbEntry {
+    pub iss: String,
```

Comment:
Won't repeating the `iss` be inefficient? Wouldn't you want a table for each `iss`?
As well as a table for each `aud`?

Reply:
i think we don't need them in a document-oriented DB.
If later we found our query is slow, we can create specific Firestore indexes on top of the raw documents.

https://cloud.google.com/firestore/docs/concepts/index-overview